drivers/spi/spi_mcux_lpspi.c

```diff
@@ -226,7 +215,6 @@ static int spi_mcux_configure(const struct device *dev, const struct spi_config
 	master_config.pinCfg = config->data_pin_config;
 
 	LPSPI_MasterInit(base, &master_config, clock_freq);
-	LPSPI_MasterTransferCreateHandle(base, &data->handle, spi_mcux_master_callback, data);
```

Comment:
minor nit on commit message:

"creation outide of the configure call" -- outside instead of outide